packages/mattermost-redux/src/action_types/worktemplates.ts

```diff
@@ -4,6 +4,13 @@
 import keyMirror from 'mattermost-redux/utils/key_mirror';
 
 export default keyMirror({
-    RECEIVED_DEV_WORKTEMPLATES: null,
-});
+    WORK_TEMPLATE_CATEGORIES_REQUEST: null,
+    RECEIVED_WORK_TEMPLATE_CATEGORIES: null,
+    WORK_TEMPLATE_CATEGORIES_SUCCESS: null,
+    WORK_TEMPLATE_CATEGORIES_FAILURE: null,
 
+    WORK_TEMPLATES_REQUEST: null,
+    RECEIVED_WORK_TEMPLATES: null,
+    WORK_TEMPLATES_SUCCESS: null,
+    WORK_TEMPLATES_FAILURE: null,
```

Comment:
I am sorry i know this is not ready for review yet. But i just wanted to add that do you plan on using these network request action such as `WORK_TEMPLATES_SUCCESS, WORK_TEMPLATES_FAILURE` request. If yes then i would encourage you to please find a way to having loading indication in the component itself. If not then feel free to remove it. Because going forward we are planning to get rid of all the intermediate action requests. https://mattermost.atlassian.net/browse/MM-13432

Reply:
Thanks for heads-up. @M-ZubairAhmed